src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+$card-width: 200px;
+$card-font: roboto, helvetica, sans-serif;
+$card-color: #060b35;
+$card-buy-color: #00acdc;
+
+.card {
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 17px;
+  font-family: $card-font;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    max-width: $card-width -40px;
```

Comment:
Consider using `width` instead of `max-width` to set a fixed width for the image.